addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/OpenApiAPI.java

```diff
@@ -71,42 +75,47 @@ public String getPrefix() {
     @Override
     public ApiResponse handleApiAction(String name, JSONObject params) throws ApiException {
         if (ACTION_IMPORT_FILE.equals(name)) {
-            File file = new File(params.getString(PARAM_FILE));
```

Comment:
This needs to be reverted, now all errors are handled as illegal parameter for the target parameter.